src/Dapr.Actors.AspNetCore/Dapr.Actors.AspNetCore.csproj

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.9" />
```

Comment:
This should not be needed here `Microsoft.AspNetCore.App` - I'll bet you did this because the actors is referencing `3.1.9` directly - it should be referencing `3.1.0`.